protocols/v2/subprotocols/mining/src/open_channel.rs

```diff
@@ -9,41 +9,47 @@ use core::convert::TryInto;
 #[cfg(feature = "with_serde")]
 use core::convert::TryInto;
 
-/// # OpenStandardMiningChannel (Client -> Server)
-/// This message requests to open a standard channel to the upstream node.
-/// After receiving a SetupConnection.Success message, the client SHOULD respond by opening
-/// channels on the connection. If no channels are opened within a reasonable period the server
-/// SHOULD close the connection for inactivity.
-/// Every client SHOULD start its communication with an upstream node by opening a channel,
-/// which is necessary for almost all later communication. The upstream node either passes
-/// opening the channel further or has enough local information to handle channel opening on its
-/// own (this is mainly intended for v1 proxies).
-/// Clients must also communicate information about their hashing power in order to receive
-/// well-calibrated job assignments.
+/// Message used by a downstream to request opening a Standard Channel.
+///
+/// Upon receiving [`OpenStandardMiningChannelSuccess`] message, the downstream should open
+/// channel(s) on the connection within a reasonable period, otherwise the upstream should close
+/// the connection for inactivity.
```

Comment:
this phrase makes no sense

@jbesraa this seems related to that discussion we had over Discord DMs where we clarified this misconception

Reply:
original wording from spec:

> /// This message requests to open a standard channel to the upstream node.
> /// After receiving a SetupConnection.Success message, the client SHOULD respond by opening
> /// channels on the connection. If no channels are opened within a reasonable period the server
> /// SHOULD close the connection for inactivity.

Reply:
whenever we deal with specs, we should be extra careful in avoiding introducing semantic changes